src/ts/playerutils.ts

```diff
@@ -59,6 +59,29 @@ export namespace PlayerUtils {
     return player.getSeekableRange() && player.getSeekableRange().start || defaultValue;
   }
 
+  /**
+   * Returns seekable time range.
+   * For live streams it will calculate timestamp timerange if tweak
+   * `enable_seek_for_vod` is not present
```

Comment:
I don't know what `enable_seek_for_vod` means but I guess you meant `enable_seek_for_live`.